### PR TITLE
multiple products + undefined error

### PR DESCRIPTION
**Pull request by shrouti1507**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/am/transform.js [hunk 1]
```diff
@@ -462,23 +473,35 @@ function processTransaction(message) {
...
   if (
     messageType === EventType.TRACK &&
     (eventType === Event.PRODUCT_LIST_VIEWED.name ||
-      eventType === Event.PRODUCT_LIST_CLICKED)
+      eventType === Event.PRODUCT_LIST_CLICKED.name)
   ) {
-    toSendEvents.push(processProductListAction(message));
+    // toSendEvents.push(processProductListAction(message));
+    temp = processProductListAction(message);
+    // eslint-disable-next-line no-plusplus
```
Comment (ruchiramoitra) [inline v0/destinations/am/transform.js:489]: don't disable these.. change i++ to I += 1 . 
for others so check what the es lint is saying search and try to replace.

File: v0/destinations/am/transform.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
Comment (arnab-p) [inline v0/destinations/am/transform.js:1]: Remove `no-console`

File: v0/destinations/am/transform.js [hunk 3]
```diff
@@ -450,10 +452,19 @@ function processTransaction(message) {
 
   // Why are we skipping Checkout started and Order Updated?
   // Order Cancelled is not being sent, but we are handling it here
-  if (
-    eventType === EventType.ORDER_CANCELLED.name ||
-    eventType === EventType.ORDER_REFUNDED.name
-  ) {
+  if (eventType === Event.ORDER_CANCELLED.name) {
+    // eslint-disable-next-line no-undef
```
Comment (arnab-p) [inline v0/destinations/am/transform.js:456]: Remove line-level `eslint-disable`

File: v0/destinations/am/transform.js [hunk 4]
```diff
@@ -462,23 +473,35 @@ function processTransaction(message) {
 function process(event) {
   const respList = [];
   const { message, destination } = event;
+
   const messageType = message.type.toLowerCase();
   const eventType = message.event ? message.event.toLowerCase() : undefined;
+
   const toSendEvents = [];
+  let temp;
```
Comment (arnab-p) [inline v0/destinations/am/transform.js:481]: Use a meaningful name for variables.

**Review by arnab-p — CHANGES_REQUESTED** [review]

**Pull request closed without merging**